modules/ducktests/tests/docker/ducker-ignite

```diff
@@ -60,11 +60,18 @@ Usage: ${script_path} [command] [options]
 help|-h|--help
     Display this help message
 
-up [-n|--num-nodes NUM_NODES] [-f|--force] [docker-image]
-        [-C|--custom-ducktape DIR] [-e|--expose-ports ports]
+build [-j|--jdk JDK] [image-name]
+    Build an docker image that ducktape uses to up cluster. Image is tagged with specified ${image_name}.
```

Comment:
s/an docker image/a docker image/ 
And is it correct that ducktape uses image? It uses a bunch of running containers, based on this image.
I think, that "Build a docker image for running tests." is a little bit better.

Reply:
I see such pipeline: build - up - run. Tests are running on containers on final stage while image is used on "up" stage. I think I should make replacement to make it clearer. Other docs use term "node", so let's use it too:
> docker image that ducktape uses to up cluster.
to
> docker image that represents a ducker node.

Reply:
Please check, is it ok now?